cli/build-public-library.js

```diff
@@ -91,7 +94,21 @@ function writeTSConfig() {
         ]
       }
     },
-    'files': getEntryPointFiles()
+    'files': getEntryPointFiles(),
+    'exclude': [
+      'node_modules',
+      '**/*.spec.ts',
+      '**/*.e2e-spec.ts'
```

Comment:
Should we also exclude `pact-spec.ts` files?

Reply:
Sure, why not.